account_banking_nl_ing_mt940/account_banking_nl_ing_mt940.py

```diff
@@ -20,36 +21,41 @@
 ##############################################################################
 import re
 from openerp.tools.translate import _
-from openerp.addons.account_banking.parsers.models import (
-    parser,
-    mem_bank_transaction,
-)
+from openerp.addons.account_banking.parsers.models import parser,\
+    mem_bank_transaction
 from openerp.addons.account_banking_mt940.mt940 import MT940, str2float
 
 
-class transaction(mem_bank_transaction):
+class Transaction(mem_bank_transaction):
+    """Override validity check in base class."""
     def is_valid(self):
-        '''allow transactions without remote account'''
+        """allow transactions without remote account"""
         return bool(self.execution_date) and bool(self.transferred_amount)
 
 
 class IngMT940Parser(MT940, parser):
+    """Define mt940 parser for Dutch ING bank."""
     name = _('ING MT940 (structured)')
     country_code = 'NL'
     code = 'INT_MT940_STRUC'
+    footer_regex = '^-}$'
 
     tag_61_regex = re.compile(
-        r'^(?P<date>\d{6})(?P<sign>[CD])(?P<amount>\d+,\d{2})N(?P<type>.{3})'
-        r'(?P<reference>\w{1,16})')
+        r'^(?P<date>\d{6})(?P<line_date>\d{4})'
```

Comment:
Also from the document above, I don't see a line date there. You probably used some real test file that contains this extra field, would it be an option to make it optional in the regex (`\d{0,4}`)? I doesn't seem to be used anyways